inference-engine/include/cpp/ie_memory_state.hpp

```diff
@@ -75,4 +80,5 @@ class MemoryState {
     }
 };
 
+using VariableState = MemoryState;
```

Comment:
can we rename `MemoryState` to `VariableState` and make the following alias:
```c++
/*
 * @brief For compatibility reasons.
 */
using MemoryState = VariableState;
```

Reply:
ok